app/src/main/java/project/pb/fragments/FAQFragment.java

```diff
@@ -34,69 +37,39 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
+                "Is the food available at the Open day?", "What time does the Open day usually end?"
+        );
 
-        List<String> answer1 = new ArrayList<>();
-        List<String> answer2 = new ArrayList<>();
-        List<String> answer3 = new ArrayList<>();
-        List<String> answer4 = new ArrayList<>();
-        List<String> answer5 = new ArrayList<>();
-        List<String> answer6 = new ArrayList<>();
-        List<String> answer7 = new ArrayList<>();
-        List<String> answer8 = new ArrayList<>();
-        List<String> answer9 = new ArrayList<>();
-        List<String> answer10= new ArrayList<>();
-        List<String> answer11 = new ArrayList<>();
-        List<String> answer12 = new ArrayList<>();
+        answers_completed = Arrays.asList(
+                "Yes, you can find paid parking outside.", "The school is located near the station Beurs",
+                "Yes, snacks and drinks are provided", "Yes, you can find paid parking outside.",
+                "The school is located near the station Beurs",
+                "Yes, snacks and drinks are provided", "Depends on the chosen study but usually at 15:00",
+                "Depends on the chosen study but usually at 15:00", "Yes, you can find paid parking outside.",
+                "The school is located near the station Beurs", "Yes, snacks and drinks are provided",
+                "Depends on the chosen study but usually at 15:00"
+        );
 
-        answer1.add("Yes, you can find paid parking outside.");
-        answer2.add("The school is located near the station Beurs");
-        answer3.add("Yes, snacks and drinks are provided");
-        answer4.add("Depends on the chosen study but usually at 15:00");
-        answer5.add("Yes, you can find paid parking outside.");
-        answer6.add("The school is located near the station Beurs");
-        answer7.add("Yes, snacks and drinks are provided");
-        answer8.add("Depends on the chosen study but usually at 15:00");
-        answer9.add("Yes, you can find paid parking outside.");
-        answer10.add("The school is located near the station Beurs");
-        answer11.add("Yes, snacks and drinks are provided");
-        answer12.add("Depends on the chosen study but usually at 15:00");
-
-        answers.put(Questions.get(0),answer1);
-        answers.put(Questions.get(1),answer2);
-        answers.put(Questions.get(2),answer3);
-        answers.put(Questions.get(3),answer4);
-        answers.put(Questions.get(4),answer5);
-        answers.put(Questions.get(5),answer6);
-        answers.put(Questions.get(6),answer7);
-        answers.put(Questions.get(7),answer8);
-        answers.put(Questions.get(8),answer9);
-        answers.put(Questions.get(9),answer10);
-        answers.put(Questions.get(10),answer11);
-        answers.put(Questions.get(11),answer12);
+        for (int i = 0; i < questions.size(); i++) {
+            answers.put(questions.get(i), Arrays.asList(answers_completed.get(i)));
```

Comment:
```suggestion
            answers.put(questions.get(i), Arrays.asList(answers_completed.get(i)));
```
nice way to do it, never thought about this, goodjob